src/simulation/openables/ElvenCrystalChest.ts

```diff
@@ -0,0 +1,174 @@
+import LootTable from '../../structures/LootTable';
+import SimpleOpenable from '../../structures/SimpleOpenable';
+
+/* Dragonstone armour roll */
+const DragonStoneTable = new LootTable()
+	.add('Dragonstone full helm', 1, 1)
+	.add('Dragonstone platebody', 1, 1)
+	.add('Dragonstone platelegs', 1, 1)
+	.add('Dragonstone gauntlets', 1, 1)
+	.add('Dragonstone boots', 1, 1);
+
+const ElvenCrystalChestTable = new LootTable()
+	.oneIn(10000, 'Uncut onyx')
+	.oneIn(500, DragonStoneTable)
+	.add(
+		[
+			['Uncut dragonstone', 1],
+			['Tooth half of key', 1],
+			['Coins', [10000, 15000]]
+		],
+		1,
+		32
+	)
+	.add(
+		[
+			['Uncut dragonstone', 1],
+			['Loop half of key', 1],
+			['Coins', [10000, 15000]]
+		],
+		1,
+		32
+	)
+	.add(
+		[
+			['Uncut dragonstone', 1],
+			['Uncut ruby', [10, 13]],
+			['Uncut diamond', [5, 8]]
+		],
+		1,
+		32
+	)
+	.add(
+		[
+			['Uncut dragonstone', 1],
+			['Crystal key', 1]
+		],
+		1,
+		24
+	)
+	.add(
+		[
+			['Uncut dragonstone', 1],
+			['Coins', [30000, 50000]],
+			['Crystal shard', [8, 13]]
+		],
+		1,
+		20
+	)
+	.add(
+		[
+			['Uncut dragonstone', 1],
+			['Crystal shard', [20, 30]]
+		],
+		1,
+		17
+	)
+	.add(
+		[
+			['Uncut dragonstone', 1],
+			['Crystal shard', [4, 6]],
+			['Rune platelegs', 1]
```

Comment:
same as reg crystal chest

Reply:
dosen't affect inflation here since the odds are right, resolving all

Reply:
original review still stands